### ISSUE TITLE
OLD_CONCEPT compatibility broken in 4.3.x?

### ISSUE THREAD

**Issue by jsanchezfr** [52North/SOS#505]
Using OLD_CONCEPT configuration fails. In 4.2.0 goes ok,
Looking for the problem, I started a new DB from scratch, with the configuration:
OLD-CONCEPT
Using TRANSACTIONAL

With no data, the CACHE updates OK (It would be strange if not).

I've populated some tables with data with a pair of data (2 types of precedure/offering) using MySQL Workbench:
```
codespace:

# codespaceId, codespace
1, http://www.opengis.net/def/nil/OGC/0/unknown

featureofinteresttype:
# featureOfInterestTypeId, featureOfInterestType
1, http://www.opengis.net/def/samplingFeatureType/OGC-OM/2.0/SF_SamplingPoint

proceduredescriptionformat:
# procedureDescriptionFormatId, procedureDescriptionFormat
20953, http://www.opengis.net/sensorML/1.0.1

procedure:
# procedureId, hibernateDiscriminator, procedureDescriptionFormatId, identifier, codespace, name,
1, T, 20953, Nivel_de_agua, 1, , , , F, F, , F
2, T, 20953, Caudal_de_agua, 1, , , , F, F, , F

offering:
# offeringId, hibernateDiscriminator, identifier, codespace, name, codespaceName, description, disabled
1, T, Nivel_de_agua, , Nivel de agua, , , F
2, T, Caudal_de_agua, , Caudal de agua, , , F

observationtype:
# observationTypeId, observationType
20953, http://www.opengis.net/def/observationType/OGC-OM/2.0/OM_Measurement

observationconstellation:
# observationConstellationId, observablePropertyId, procedureId, observationTypeId, offeringId, deleted, hiddenChild
1, 1, 1, 20953, 1, F, F
2, 2, 2, 20953, 2, F, F

observableproperty:
# observablePropertyId, hibernateDiscriminator, identifier, codespace, name, codespaceName, description, disabled
1, T, water_discharge, 1, caudal, 1, Caudal de agua, F
2, T, water_level, 1, altura, 1, Altura del agua, F

unit:
# unitId, unit
1, m
2, m3/s
```
The data seems correct. But the cache rebuild fail:
```
2017-02-07 11:46:18,106 TRACE [Thread-14] [ContentCacheControllerImpl.java:342] Finished update org.n52.sos.cache.ctrl.action.CompleteCacheUpdate@127b3ff
2017-02-07 11:46:18,106 DEBUG [Thread-14] [ContentCacheControllerImpl.java:302] State change: RUNNING -> FAILED
2017-02-07 11:46:18,106 WARN [Thread-14] [ContentCacheControllerImpl.java:347] Update failed!
org.n52.sos.ogc.ows.CompositeOwsException: Implemented series DAO is missing!
Implemented series DAO is missing!
        at org.n52.sos.ds.hibernate.SosCacheFeederDAO.updateCache(SosCacheFeederDAO.java:107) ~[hibernate-dao-4.3.9.jar:na]
        at org.n52.sos.cache.ctrl.action.CompleteCacheUpdate.execute(CompleteCacheUpdate.java:45) ~[cache-4.3.9.jar:na]
        at org.n52.sos.cache.ctrl.ContentCacheControllerImpl$CompleteUpdate.execute(ContentCacheControllerImpl.java:341) [cache-4.3.9.jar:na]
        at org.n52.sos.cache.ctrl.ContentCacheControllerImpl$CompleteUpdate.execute(ContentCacheControllerImpl.java:331) [cache-4.3.9.jar:na]
        at org.n52.sos.cache.ctrl.ContentCacheControllerImpl.runCurrent(ContentCacheControllerImpl.java:132) [cache-4.3.9.jar:na]
        at org.n52.sos.cache.ctrl.ContentCacheControllerImpl.executeComplete(ContentCacheControllerImpl.java:182) [cache-4.3.9.jar:na]
        at org.n52.sos.cache.ctrl.ContentCacheControllerImpl.update(ContentCacheControllerImpl.java:120) [cache-4.3.9.jar:na]
        at org.n52.sos.cache.ctrl.ContentCacheControllerImpl.update(ContentCacheControllerImpl.java:219) [cache-4.3.9.jar:na]
        at org.n52.sos.web.admin.AbstractAdminController$1.run(AbstractAdminController.java:52) [admin-controller-4.3.9.jar:na]
        at java.lang.Thread.run(Thread.java:745) [na:1.7.0_80]
Caused by: org.n52.sos.exception.ows.NoApplicableCodeException: Implemented series DAO is missing!
        at org.n52.sos.ds.hibernate.dao.DaoFactory.getSeriesDAO(DaoFactory.java:87) ~[hibernate-common-4.3.9.jar:na]
        at org.n52.sos.ds.hibernate.cache.base.ProcedureCacheUpdateTask.getProcedureInformationFromDbAndAddItToCacheMaps(ProcedureCacheUpdateTask.java:70) ~[hibernate-dao-4.3.9.jar:na]
        at org.n52.sos.ds.hibernate.cache.base.ProcedureCacheUpdateTask.execute(ProcedureCacheUpdateTask.java:91) ~[hibernate-dao-4.3.9.jar:na]
        at org.n52.sos.util.RunnableAction.run(RunnableAction.java:55) ~[api-4.3.9.jar:na]
        at org.n52.sos.util.ThreadableAction.run(ThreadableAction.java:50) ~[api-4.3.9.jar:na]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471) ~[na:1.7.0_80]
        at java.util.concurrent.FutureTask.run(FutureTask.java:262) ~[na:1.7.0_80]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) ~[na:1.7.0_80]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) ~[na:1.7.0_80]
        ... 1 common frames omitted 
```

**Comment by EHJ-52n** [comment]
@jsanchezfr 

Thank you for your issue/feedback. I updated the layout a bit and hope you are okay with these changes.
Kind regards!

**Comment by CarstenHollmann** [comment]
Duplicate of #503.

The fix is contained in the [v4.3.10](https://github.com/52North/SOS/releases/tag/v4.3.10) release.

**Comment by jsanchezfr** [comment]
Sorry for the duplicate.
I've download version 4.3.10 master, compiled ok, and when creating, in the last step:

[error.txt](https://github.com/52North/SOS/files/766446/error.txt)

Using MySQL, OLD_CONCEPT configuration, blank DB.
